### ISSUE TITLE
Bunch-beta not showing bunches menu Monterey M2 Air

### ISSUE THREAD

**Issue by stuzog** [ttscoff/bunch#259]
Bunch Beta 1.4.12 does not show the bunches drop-down folders/files menu in Monterey 12.6 on M2 MacBook Air. Bunch 1.4.11 works properly. Both apps have Full Disk Access in System Preferences -> Privacy.

**Comment by ttscoff** [comment]
Do you mean your Bunches don't show up at all?

-Brett

**Comment by stuzog** [comment]
Yes, exactly. I have allowed both bunch and bunch-beta Full Disk Access.

**Comment by ttscoff** [comment]
Possibly related, I'm running into all kinds of trouble with the beta this morning and will need to figure out what broke there. I'll try to put out an update soon, but recommend using the stable version until then.

**Comment by stuzog** [comment]
Same issue with bunch Version 1.4.17 (180)
![Screenshot 2024-11-22 at 8 35 55 PM](https://github.com/user-attachments/assets/ed138605-8f8a-41c5-b41e-1ab1f960c0a1)
![Screenshot 2024-11-22 at 8 35 01 PM](https://github.com/user-attachments/assets/af3430a0-eb3b-4872-9173-1a3323562c9b)

**Comment by ttscoff** [comment]
Are you nesting folders more than one deep? Because that won't work.

**Comment by stuzog** [comment]
Yes, that's it, these folders were too deep. Is there any way the folder depth could be increased? Otherwise, I'm going to have a very long bunch sub-menu.

**Comment by ttscoff** [comment]
Due to the way I originally implemented subfolders, there's a hard limit of 1 level. Changing that would be a *major* refactor.

**Comment by ttscoff** [comment]
Have you considered building a menu dialog into a master bunch for the presets? One bunch that offered a popup selection of which snippet to load? Without seeing the contents of all of these bunches, their similarity seems ripe for implementing bunch's interactive features
